swagger.js

```diff
@@ -0,0 +1,20 @@
+const swaggerJSDoc = require('swagger-jsdoc');
+const swaggerUi = require('swagger-ui-express');
+
+const swaggerDefinition = {
+  info: {
+    title: 'API Documentation',
+    version: '1.0.0',
+    description: 'API documentation for Ella`s Node.js application',
+  },
+  basePath: '/',
+};
+
+const options = {
+  swaggerDefinition,
+  apis: ['./routes/*.js'],
```

Comment:
Is this the correct file path? Swagger.js file is outside the src folder.

Reply:
I have changed the file path, thank you